### PR TITLE
#3280 Rename streamApp to stream in the UI

### PR DESCRIPTION
**Pull request by konekoya**
### Checklist
- [x] [Review Contribution Guidelines](https://ohara.readthedocs.io/en/latest/contributing.html)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#ohara-team))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #3280)
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

**Comment by chia7712** [comment]
有些APIs上也有"streamapp"等類似的字眼 要麻煩你一起在這隻PR改掉喔~不然APIs測試會噴掉

**Comment by konekoya** [comment]
> 有些APIs上也有"streamapp"等類似的字眼 要麻煩你一起在這隻PR改掉喔~不然APIs測試會噴掉

No problem. I have done a global "search and replace". Let's see if everything can still pass as it should :)

**Comment by konekoya** [comment]
Hmm, looks like there's still some in the backend code, will update PR again...

**Comment by chia7712** [comment]
> Hmm, looks like there's still some in the backend code, will update PR again...

:)

File: ohara-streams/src/main/java/com/island/ohara/streams/config/StreamDefUtils.java [hunk 1]
```diff
@@ -158,7 +158,7 @@
           .blacklist(new HashSet<>(Arrays.asList("stop", "start", "pause", "resume")))
           .build();
 
-  public static final String STREAM_APPLICATION = "streamApp";
+  public static final String STREAM_APPLICATION = "stream";
```
Comment (konekoya) [inline ohara-streams/src/main/java/com/island/ohara/streams/config/StreamDefUtils.java]: @chia7712 should I rename this from _STREAM_APPLICATION_ to _STREAM_?

Comment (chia7712) [reply ohara-streams/src/main/java/com/island/ohara/streams/config/StreamDefUtils.java]: U are right :)

**Comment by chia7712** [comment]
retry configurator

**Comment by chia7712** [comment]
retry configurator

**Comment by konekoya** [comment]
Thanks for the help @chia7712 ❤️

**Pull request merged**